test/plugin/scenarios/dubbo-2.7.x-scenario/bin/startup.sh

```diff
@@ -18,4 +18,5 @@
 
 home="$(cd "$(dirname $0)"; pwd)"
 
-java -jar ${agent_opts} ${home}/../libs/dubbo-2.7.x-scenario.jar &
\ No newline at end of file
+java -Dskywalking.plugin.dubbo.collect_consumer_address=true -Dskywalking.plugin.dubbo.collect_consumer_arguments=true \
```

Comment:
Do you forget to remove this? `skywalking.plugin.dubbo.collect_consumer_address=true`.

Reply:
fixed. please recheck